screens/ChatScreen.js

```diff
@@ -82,23 +85,24 @@ class ChatScreen extends React.Component {
                             {messages &&
                                 messages.map(
                                     ({
-                                        message,
+                                        text,
                                         id,
-                                        isSelf,
-                                        user
+                                        users_id,
+                                        created_at
                                     }) => (
                                         <ChatMessage
-                                            message={message}
+                                            text={text}
                                             key={id}
-                                            isSelf={isSelf}
-                                            user={user}
+                                            users_id={users_id}
```

Comment:
can you change these to match convention which is camelCase?

Reply:
Whoops, sure thing.